README.md

```diff
@@ -93,6 +98,7 @@ To test your changes on a VM:
 
 ### Releasing
 To release a new version, run the [release pipeline](https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_puppetserver-ca_init-multijob_main/), which will bump the version, tag, build, and release the gem.
+To release a new version, run the [release pipeline](https://jenkins-platform.delivery.puppetlabs.net/job/platform_puppetserver-ca_init-multijob_1.x/), which will bump the version, tag, build, and release the gem.
```

Comment:
We only want this sentence once, but with the URL `https://jenkins-platform.delivery.puppetlabs.net/job/platform_puppetserver-ca_init-multijob_main/`, which contains both the new, correct `jenkins-platform` and the right branch, `main`.

Reply:
Oh. I wasn't sure what to do there when solving the conflicts.

Reply:
+1

I thought about making this change myself, but I wanted to give @Tu2607 the pleasure of resolving merge conflicts 😜

Reply:
lol. I see how it is 😆

Reply:
yeah, you kinda have to just use your judgment when resolving merge conflicts, but usually you pick one version or the other.

The most common kinds we see are version bumps, which we usually resolve by maintaining the state of the current branch (e.g. keep puppet6-ish versions on the `6.x` branch and puppet7-ish versions on the `main` branch), or things like this where the spirit of the change should be merged up, but requires some manual intervention due to differing versions (i.e. keep the base url change, but keep the distinction between the `1.x` and `main` jobs).

Reply:
Okay. I'll remove that passage.  Thanks for the heads up guys!

Reply:
so yeah, we didn't want to just remove this altogether. we still want to change `jenkins-master-prod-1` to `jenkins-platform`, but keep the `_main` version of the job instead of taking the `_1.x` version from the other branch.

Reply:
sorry if my comment was confusing

Reply:
I hope the new commit fix that :)